ci/Ios.Jenkinsfile

```diff
@@ -99,10 +93,11 @@ pipeline {
 							generateXCodeProjects()
 							util.runFastlane("de.tutao.tutanota", "adhoc_prod")
 							if (params.RELEASE) {
-								writeReleaseNotesForAppStore()
-								util.runFastlane("de.tutao.tutanota", "appstore_prod submit:true")
+								util.runFastlane("de.tutao.tutanota", "build_mail_prod")
```

Comment:
shouldn't we build both appstore and adhoc one, like for staging few lines above?